htop.c

```diff
@@ -268,6 +289,46 @@ static void setCommFilter(State* state, char** commFilter) {
    *commFilter = NULL;
 }
 
+#ifdef HAVE_LIBCAP
+static int dropCapabilities(void) {
+   /* capabilities we keep to operate */
+   const cap_value_t keepcaps[] = {
+      CAP_DAC_READ_SEARCH,
+      CAP_SYS_PTRACE,
+   };
```

Comment:
Check for further capabilities we might need.

On first glance:
* CAP_KILL
* CAP_SYS_NICE